diff -up gwenview-20.12.3/app/org.kde.gwenview.desktop.cmake.omv~ gwenview-20.12.3/app/org.kde.gwenview.desktop.cmake
diff -up gwenview-20.12.3/CMakeLists.txt.omv~ gwenview-20.12.3/CMakeLists.txt
--- gwenview-20.12.3/CMakeLists.txt.omv~	2021-03-18 22:32:31.187714174 +0100
+++ gwenview-20.12.3/CMakeLists.txt	2021-03-18 22:32:56.484004611 +0100
@@ -166,6 +166,8 @@ set(IMAGE_MIME_TYPES_LIST
     image/x-webp
     image/webp
     image/x-tga
+    image/avif
+    image/jxl
     application/x-krita
 )
 if (KF5KDcraw_FOUND)
