diff -urN gwenview-4.10.2/lib/document/document.cpp gwenview-4.10.2-patched/lib/document/document.cpp
--- gwenview-4.10.2/lib/document/document.cpp	2013-05-06 08:57:46.774454187 +1100
+++ gwenview-4.10.2-patched/lib/document/document.cpp	2013-05-06 08:58:20.057037432 +1100
@@ -387,9 +387,6 @@
         return false;
     } else if (loadingState() == Loaded) {
         // Resample image from the full one
-        if (isBusy()) {
-            return false;
-        }
         d->mDownSampledImageMap[invertedZoom] = d->mImage.scaled(d->mImage.size() / invertedZoom, Qt::KeepAspectRatio, Qt::FastTransformation);
         if (d->mDownSampledImageMap[invertedZoom].size().isEmpty()) {
             d->mDownSampledImageMap[invertedZoom] = d->mImage;
