diff -urN gwenview-4.11.0/app/gwenview.desktop gwenview-4.11.0-patched/app/gwenview.desktop
--- gwenview-4.11.0/app/gwenview.desktop	2013-07-14 21:02:25.000000000 +1100
+++ gwenview-4.11.0-patched/app/gwenview.desktop	2013-08-16 23:41:28.676318084 +1100
@@ -171,7 +171,7 @@
 Icon=gwenview
 Type=Application
 Categories=Qt;KDE;Graphics;Viewer;Photography;
-MimeType=inode/directory;image/gif;image/jpeg;image/png;image/bmp;image/x-eps;image/x-ico;image/x-portable-bitmap;image/x-portable-graymap;image/x-portable-pixmap;image/x-xbitmap;image/x-xpixmap;image/tiff;image/x-psd;image/x-webp;
+MimeType=image/gif;image/jpeg;image/png;image/bmp;image/x-eps;image/x-ico;image/x-portable-bitmap;image/x-portable-graymap;image/x-portable-pixmap;image/x-xbitmap;image/x-xpixmap;image/tiff;image/x-psd;image/x-webp;image/svg+xml;image/svg+xml-compressed;
 X-DocPath=gwenview/index.html
 # InitialPreference should be greater than Okular so that Gwenview is the
 # primary application associated with images, but less than Konqueror or Dolphin
