diff -urN gwenview-4.10.0/app/gwenview.desktop gwenview-4.10.0-patched/app/gwenview.desktop
--- gwenview-4.10.0/app/gwenview.desktop	2013-01-31 07:34:46.000000000 +1100
+++ gwenview-4.10.0-patched/app/gwenview.desktop	2013-02-11 19:19:45.254012836 +1100
@@ -166,7 +166,7 @@
 Icon=gwenview
 Type=Application
 Categories=Qt;KDE;Graphics;Viewer;Photography;
-MimeType=inode/directory;image/gif;image/jpeg;image/png;image/bmp;image/x-eps;image/x-ico;image/x-portable-bitmap;image/x-portable-graymap;image/x-portable-pixmap;image/x-xbitmap;image/x-xpixmap;image/tiff;
+MimeType=image/gif;image/jpeg;image/png;image/bmp;image/x-eps;image/x-ico;image/x-portable-bitmap;image/x-portable-graymap;image/x-portable-pixmap;image/x-xbitmap;image/x-xpixmap;image/tiff;
 X-DocPath=gwenview/index.html
 # InitialPreference should be greater than Okular so that Gwenview is the
 # primary application associated with images, but less than Konqueror or Dolphin
