diff -up gwenview-18.12.0/lib/jpegcontent.cpp.omv~ gwenview-18.12.0/lib/jpegcontent.cpp
--- gwenview-18.12.0/lib/jpegcontent.cpp.omv~	2018-12-24 02:25:33.652526830 +0100
+++ gwenview-18.12.0/lib/jpegcontent.cpp	2018-12-24 02:25:42.264658838 +0100
@@ -44,6 +44,7 @@ extern "C" {
 // Exiv2
 #include <exiv2/exif.hpp>
 #include <exiv2/image.hpp>
+#include <exiv2/version.hpp>
 
 // Local
 #include "jpegerrormanager.h"
